clippy_lints/src/redundant_clone.rs

```diff
@@ -146,7 +145,7 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for RedundantClone {
                 // `arg` is a reference as it is `.deref()`ed in the previous block.
                 // Look into the predecessor block and find out the source of deref.
 
-                let ps = mir_read_only.predecessors_for(bb);
+                let ps = mir.predecessors_for(bb);
```

Comment:
FYI, `predecessors_for` is going away soon as well. The replacement would be:

```suggestion
                let ps = mir.predecessors()[bb];
```

Reply:
Whoops. Forgot a borrow.

Reply:
I added the `&`.